### PR TITLE
Fix simulation (Kick and initial fall) 

### PR DESCRIPTION
**Pull request by Flova**
# Summary
Currently, we are no able to kick and fall down frequently in the simulation. This fixes that by adding a different startup animation and slightly different kick animation. 

This needs to be tested on the real robot and we might need different animations for sim and real.

## Checklist

- [x] Run `colcon build`
- [x] Write documentation
- [x] Test on your machine
- [ ] Test on the robot
- [ ] Create issues for future work
- [x] Triage this PR and label it

### CODE REVIEW

File: bitbots_wolfgang/wolfgang_animations/animations/kick/kick_left.json [hunk 1]
```diff
@@ -36,7 +36,7 @@
                 "LHipYaw": -1.14,
                 "LKnee": 56.51,
                 "RAnklePitch": 35.86,
-                "RAnkleRoll": -5.36,
+                "RAnkleRoll": 5.36,
```
Comment (val-ba) [inline bitbots_wolfgang/wolfgang_animations/animations/kick/kick_left.json:39]: How do you know it is positive? Did the left and right ankles get mixed up? Or is that the change you made to the kick?

Comment (Flova) [reply bitbots_wolfgang/wolfgang_animations/animations/kick/kick_left.json:39]: This is the change I made for the kick to work in sim

**Review by val-ba — APPROVED** [review]
I tested this in simulation, and the robot does not fall in the beginning anymore and it also performed kicks without falling.

**Comment by Flova** [comment]
This was not intended to be merged without testing the changes also on the real robot.

**Comment by Flova** [comment]
But I might test them later today

**Pull request merged**